piecrust/src/store/tree.rs

```diff
@@ -4,7 +4,10 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use std::collections::{BTreeMap, BTreeSet};
+use std::{
+    cell::Ref,
+    collections::{BTreeMap, BTreeSet},
+};
```

Comment:
Remove this nested use

Reply:
We use it in other parts of the code here in piecrust, and I tend to prefer it, so I think I'll keep it.